janusgraph-core/src/main/java/org/janusgraph/core/TransactionBuilder.java

```diff
@@ -138,6 +138,14 @@ public interface TransactionBuilder {
      */
     TransactionBuilder commitTime(Instant instant);
 
+    /**
+     * Enables/disables JanusGraph database level cache access during read operations.
+     * Doesn't have any effect if database level cache was disabled during JanusGraph instance creation.
+     *
+     * @param enabled Enable db-cache access during read operations
```

Comment:
The doc here is a bit ambiguous IMO. "Enable db-cache access during read operations" implies when this parameter is true, db-cache is enabled, while the fact is the opposite. Maybe we can simply change the method signature to `skipDBCacheRead()` just like `disableBatchLoading()` method?